packages/api-docs-builder/ApiBuilders/HookApiBuilder.ts

```diff
@@ -560,7 +567,7 @@ export default async function generateHookApi(hooksInfo: HookInfo, project: Type
       });
       return node;
     },
-    defaultHandlers.concat(muiDefaultParamsHandler),
+    defaultHandlers,
```

Comment:
Apparently, this didn't do anything useful. The output is the same without this handler.